### DISCUSSION TITLE
Request for new icon and offering help

### DISCUSSION THREAD

**Discussion by ppsilva** [zim-desktop-wiki/zim-desktop-wiki#2050] [Show and tell]
I really enjoy Zim desktop wiki and I would like to help but I'm a designer not a programmer

What if we could have a new icon for Zim desktop wiki? One  with less elements in order to improve readability at smaller sizes and as a plus to help recognize the software, be in the that in the system tray, dash, dock etc.

So this is something I could help with.

Here is a first quick draft/concept to start the ball rolling. I just tried to focused on the initial Z (related to typing, typography, etc) together with the idea of connecting the dots (diagrams, links, pages, tags). I tried to upload the svg but it seems GitHub does not allow uploads of that format via issues.

![zim48-pedrosilva-concept](https://user-images.githubusercontent.com/7551706/58321353-35e79980-7e1e-11e9-93a4-749fbca6932d.png)

Thanks for all your work!!

**Comment by ppsilva** [comment]
it would be great to get some feedback and also to know what would be the best way to progress with it. For example should we keep discuss in this thread and iterating from there by adding comments etc or perhaps create a new branch for this? Let me know :)

**Comment by jaap-karssenberg** [comment]
Thanks for the offer but for the moment I'm happy with the icon that we
have. I'm open to improvements but I would like to stick very close to the
current one.

An other area where as a designer you might have is about is the layout and
styling of the application itself. Since theming in gtk3 is driven by css
it might be feasible to try your hand at pushing the GUI elements.

Regards,

Jaap

Op vr 24 mei 2019 17:19 schreef ppsilva <notifications@github.com>:

**Comment by ppsilva** [comment]
sure, I understand. So maybe if I have time I can think about some small tweaks that would improve it while maintaining the overall design.

That would be really interesting :) and I am really comfortable with css. I just never styled a GTK app by myself (meaning I never did the setup, I just worried about css, colors, svgs. etc)

Oh and BTW thanks for all your work :)

**Comment by ppsilva** [comment]
![zim-icon-draft](https://user-images.githubusercontent.com/7551706/58385989-7f401080-7ff9-11e9-9e3d-acabd19faad9.png)
here is something we could maybe use at small sizes but yeah I'm not really happy with it :(, but it was what I manage while trying to keep as much as I could from the current icon. I will take a look at how to style GTK apps (setup/build for Zim etc) once I have time. I'll communicate back in a week or two :+1:

**Comment by sojusnik** [comment]
While I also think that the current Zim icon could be simplified to look more modern and scalable, improving the layout and styling would be more important. A good starting point could be tweaking the [path and bookmarks bar](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/617), especially their height in comparison to the sidebar (see content and tags tab), as shown [here](https://i.imgur.com/n4PH48T.png). The [bottom (status?) bar](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/618) could also be tweaked.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/36744029/68084619-81808380-fe51-11e9-9aad-e8ed91ca40a1.png)

I took a shot at making Zim's icon adhere to the new style of GNOME icons.

**Comment by sojusnik** [comment]
@zappedfish did you get any feedback from @jaap-karssenberg regarding your design concept? Totally love your icon and would love to see it soon as the default one!

**Comment by introt** [comment]
I've added the label instead of moving to discussions right now so this doesn't get buried right away

**Comment by dehesselle** [comment]
~Also, slightly off topic (but I want to jump at the chance for possible designer contribution), if you're looking for something with more creative freedom @zappedfish @ppsilva, you can design a background for the macOS disk image (or let's call it "installer" if you're not familiar with macOS). This is what the current one looks like:~

Edit: no longer relevant.
